### ISSUE TITLE
Retrain

### ISSUE THREAD

**Issue by rgund26** [ultralytics/ultralytics#2832]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

How to retain my old weights(which were trained by my custom dataset) with my new data set?
To increase my model accuracy.

### Additional

Now I got the new data set which needs to be trained with my old existing model. How I can retrain the model

**Comment by Kayzwer** [comment]
yolo detect train model=/path/to/.pt warmup_epochs=0 data=/path/to/.yaml

**Comment by glenn-jocher** [comment]
@rgund26 hello! To retrain your YOLOv8 model with your new dataset while keeping the existing weights, you can use the `--resume` flag in your training command. For instance, you can set `model=weights.pt` to load in your existing weights, and then use `--resume` with the path to your previous training results directory. This will allow you to resume training from where you left off with the previously trained model.

Please ensure that your new data has the same number of classes, or add the new classes at the end of your original class list and adjust accordingly in your .yaml file. It is preferable to start training the new dataset with the pre-trained weights from the old data in order to reduce training time and prevent overfitting, unless the new data has significantly different features or distributions.

Please let me know if you have any more questions or need further assistance!
